lost/src/test/java/com/mapzen/android/lost/internal/LostApiClientImplTest.java

```diff
@@ -166,4 +165,15 @@ public void isConnected_shouldReturnFalseBeforeConnected() throws Exception {
     client.disconnect();
     assertThat(client.isConnected()).isFalse();
   }
+
+  @Test public void isConnected_multipleClients_shouldReturnFalseAfterDisconnected() throws
+      Exception {
+    LostApiClient anotherClient = new LostApiClient.Builder(application).build();
+    anotherClient.connect();
+    client.connect();
+    client.disconnect();
+    assertThat(client.isConnected()).isFalse();
+    anotherClient.disconnect();
```

Comment:
Its needed so that the shared `LostClientManager` is cleaned up and the tests in `LostClientManagerTest` run as expected. Though maybe we want to refactor `LostApiClientImpl` to inject the `ClientManager` in the constructor?